legal-api/env-wf

```diff
@@ -6,22 +6,26 @@ SECRET=
 APP_SETTINGS=dev
 
 # Legal API Database
-DATABASE_USERNAME=legal
+DATABASE_USERNAME=userG5G
```

Comment:
as discussed in person - take this file out of the PR (but don't delete from repo)